tests/data/expected/parser/openapi/openapi_parser_with_query_parameters/output.py

```diff
@@ -53,9 +53,7 @@ class PetsGetParametersQuery(BaseModel):
 
 
 class PetsGetResponse(BaseModel):
-    id: int
-    name: str
-    tag: Optional[str] = None
+    __root__: List[Pet]
```

Comment:
@pvicol @hambrosia
https://github.com/koxudaxi/datamodel-code-generator/pull/1083
I found the problem in the PR.
The current implementation doesn't create an array object on response.
But it creates an item of an array.

It means the PR breaks current behavior.